services/ssp-a/src/index.js

```diff
@@ -45,13 +52,22 @@ app.use((req, res, next) => {
   next();
 });
 
-app.use(express.json());
```

Comment:
Is the import "express" then still used, or could that import be removed?

Reply:
Express is still used `const app = express()` in line 41.